red-hat-data-services/odh-manifests
# Add parameters to set an image sha256 value

For disconnected installation, odh-manifests need to receive an image link by Parameters. 
This commit changes 5 parts:
- Jupyter notebook
  - minimal-notebook-image-stream
  - generic-data-science-notebook-imagestream
- odh dashboard
- odh notebook controller
  - kf notebook controller
  - odh notebook controller

Test way:
  - Deploy odh-operator and kfdefs
    ~~~
    oc create -f https://gist.githubusercontent.com/Jooho/a269a59fe3b92da0192ae9a5b34049c5/raw/1a37c3c492a8cce8764e366c2d50feb87369fa0f/ODH-Manifest%2520test%2520deployment.yaml
    ~~~
  - Check the following objects:
    ~~~
    oc get is/s2i-generic-data-science-notebook -o yaml |grep name|grep sha
    oc get is/s2i-minimal-notebook -o yaml |grep name|grep sha
    oc get deploy rhods-dashboard -o yaml|grep 'image:'
    oc get deploy odh-notebook-controller-manager -o yaml|grep 'image:'
    oc get deploy notebook-controller-deployment -o yaml|grep 'image:'
    ~~~
  - Clean up
    ~~~
      oc delete -f https://gist.githubusercontent.com/Jooho/a269a59fe3b92da0192ae9a5b34049c5/raw/1a37c3c492a8cce8764e366c2d50feb87369fa0f/ODH-Manifest%2520test%2520deployment.yaml
    ~~~
- [x] JIRA link(s): https://issues.redhat.com/browse/RHODS-5568
